tests/integration/conftest.py

```diff
@@ -73,9 +74,9 @@ async def app(
     )
     await ops_test.model.wait_for_idle(status="active")
 
-    app_charm = await ops_test.build_charm(".")
+    charm = pytestconfig.getoption("--charm-file")
```

Comment:
Do you think it is worth making this a fixture?

Reply:
Including deploying it

Reply:
I may be misunderstanding, but isn't it already a fixture? This is part of the `app` fixture which deploys pollen and any-app and relates them.

Reply:
ah yes I missed this is in `conftest.py`. Capturing the value of command line arguments in other cases is sometimes put into a separate fixtures, up to you

Reply:
Ok, thx